lib/utils/index.js

```diff
@@ -278,11 +290,38 @@ function taskIdRegExp() {
   return new RegExp(`^([0-9a-z-]+)_([0-9]+)$`);
 }
 
-
 function getRandomInt(min, max) {
   min = Math.ceil(min);
   max = Math.floor(max);
-  return Math.floor(Math.random() * (max - min)) + min;
+  const factor = max - min === 1 ? 2 : (max - min);
+  //If we want a random of just 2 numbers then the factor must be 2, else it will always return back the lesser of two number always.
+  return Math.floor(Math.random() * (factor)) + min;
+}
+
+function getRandomCronForOnceEveryXDays(days, options) {
```

Comment:
any library available for these kinds of operations?

Reply:
Dont know :-)